packages/ember-core/src/components/mktg/header.gts

```diff
@@ -16,7 +16,7 @@ export interface MktgHeaderSignature {
 }
 
 const Header: TOC<MktgHeaderSignature> = <template>
-  <HeaderComponent ...attributes>
+  <HeaderComponent class="bg-white top-0 sticky-top w-100 gx-0" ...attributes>
```

Comment:
Can this be `bg-body` instead of `bg-white`? White never really changes meaning (unless you're in dark mode), but I think this is the purpose of the `body` color.